### PR TITLE
Private Profile and Redacted URL Set Up

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2595)

[Ticket TM-4423](https://metaphase.atlassian.net/browse/TM-4423)

### CODE REVIEW

File: talentmap_api/fsbid/views/client.py [hunk 1]
```diff
@@ -36,7 +37,11 @@ def get(self, request, pk):
         '''
         Gets a single client by perdet_seq_num
         '''
-        return Response(services.single_client(request.META['HTTP_JWT'], pk))
+        bureau_group = get_group_by_name("bureau_user")
+        is_superuser = in_superuser_group(self.request.user)
+        isBureauNotSuperuser = (bureau_group in request.user.groups.all()) and not is_superuser
+        # Option 2: we return to FE based on BE check and FE remains same
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/views/client.py:43]: 2️⃣  option 1 on FE

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/views/client.py:43]: I prefer this way

File: talentmap_api/fsbid/services/common.py [hunk 2]
```diff
@@ -47,13 +47,21 @@
...
+def get_employee_profile_urls(userid, only_redacted):
+    suffix = f"Employees/{userid}/EmployeeProfileReportByCDO"
+    suffixRedacted = f"Employees/{userid}/EmployeeProfileReportByCDO"
+
+    urls = {
+        "internalRedacted": f"{HRDATA_URL}/{suffixRedacted}",
+        "externalRedacted": f"{HRDATA_URL_EXTERNAL}/{suffixRedacted}",
     }
 
+    if not only_redacted:
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/common.py:59]: shut down to Bureau. and Post??

File: talentmap_api/fsbid/services/common.py [hunk 3]
```diff
@@ -47,13 +47,21 @@
...
-    return {
-        "internal": f"{HRDATA_URL}/{suffix}",
-        "external": f"{HRDATA_URL_EXTERNAL}/{suffix}",
+def get_employee_profile_urls(userid, only_redacted):
+    suffix = f"Employees/{userid}/EmployeeProfileReportByCDO"
+    suffixRedacted = f"Employees/{userid}/EmployeeProfileReportByCDO"
+
+    urls = {
+        "internalRedacted": f"{HRDATA_URL}/{suffixRedacted}",
+        "externalRedacted": f"{HRDATA_URL_EXTERNAL}/{suffixRedacted}",
```
Comment (M00staff) [inline talentmap_api/fsbid/services/common.py:56]: Semi unrelated question - I'm going through the code and can't find a difference between `HRDATA_URL` and `HRDATA_URL_EXTERNAL`

... am i missing something?

File: talentmap_api/fsbid/views/client.py (hunk 1 above)
Comment (M00staff) [reply talentmap_api/fsbid/views/client.py:43]: i like this option

File: talentmap_api/fsbid/services/common.py (hunk 3 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/common.py:54]: Our env variables are set on the server, based on the env - they differ there

Comment (M00staff) [reply talentmap_api/fsbid/services/common.py:54]: ahhh - see, so there's like a docker config yaml in dev, and each env 

makes sense - thanks!

**Review by M00staff — APPROVED** [review]

**Pull request merged**